<?xml version="1.0" encoding="UTF-8" ?>

<testsuite failures="1" time="0.221" errors="0" skipped="0" tests="4" name="BasicTest">
  <testcase classname="BasicTest" name="createAndRetrieveCustomer" time="0.07">
      </testcase>
  <testcase classname="BasicTest" name="tryConnectAsCustomer" time="0.034">
      </testcase>
  <testcase classname="BasicTest" name="createItem" time="0.071">
      </testcase>
  <testcase classname="BasicTest" name="ItemWorkUpdates" time="0.018">
      </testcase>
  <testcase classname="BasicTest" name="useTheWorkUpdatesRelation" time="0.028">
                  <failure type="" message="Failure, expected:&lt;1&gt; but was:&lt;5&gt;">
        <![CDATA[
          In /test/BasicTest.java, line 104
          
                  assertEquals(1, Customer.count()); :
          
        ]]>
        </failure>
        }
                  </testcase>
</testsuite>
